vcd/datasource_vcd_tm_region.go

```diff
@@ -0,0 +1,96 @@
+package vcd
+
+import (
+	"context"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func datasourceVcdTmRegion() *schema.Resource {
+	return &schema.Resource{
+		ReadContext: datasourceVcdTmRegionRead,
+
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "Region name",
+			},
+			"description": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "Region description",
+			},
+			"is_enabled": {
+				Type:        schema.TypeBool,
+				Computed:    true,
+				Description: "Defines whether the Region is enabled or not",
+			},
+			"nsx_manager_id": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "NSX-T Manager ID",
```

Comment:
To have it equal to the resource Description
```suggestion
				Description: "NSX Manager ID",
```

Reply:
Fixed